src/backend/TrafficCourts/Citizen.Service/Controllers/DisputesController.cs

```diff
@@ -562,40 +563,63 @@ private static bool IsDisputeStatus(string value, DisputeStatus status)
 
     private bool CompareNames(SubmitNoticeOfDispute message, UserInfo? user)
     {
-#if DEBUG 
+#if DEBUG
 #warning Contact Name Comparisons with BC Services Cards have been disabled 
-       return true;
+        return true;
 #endif
-        bool result = true;
 
-        // if contact type is individual then match with disputant name otherwise match with contact names
-        if (message.ContactTypeCd == DisputeContactTypeCd.INDIVIDUAL)
+        string Combine(string? name1, string? name2, string? name3)
         {
-            var givenNames = message.DisputantGivenName1
-                + (message.DisputantGivenName2 != null ? (" " + message.DisputantGivenName2) : "")
-                + (message.DisputantGivenName3 != null ? (" " + message.DisputantGivenName3) : "");
-            if (!message.DisputantSurname.Equals(user?.Surname, StringComparison.OrdinalIgnoreCase)
-                || !(message.DisputantGivenName1.Equals(user?.GivenName, StringComparison.OrdinalIgnoreCase) || givenNames.Equals(user?.GivenNames, StringComparison.OrdinalIgnoreCase)))
+            StringBuilder buffer = new StringBuilder();
+            if (!string.IsNullOrEmpty(name1))
+            {
+                buffer.Append(name1.Trim());
+            }
+
+            if (!string.IsNullOrEmpty(name2))
             {
-                result = false;
+                if (buffer.Length > 0)
+                {
+                    buffer.Append(' ');
```

Comment:
You should use double quotes for string append like: 
```suggestion
                    buffer.Append(" ");
```

Reply:
In this scenario for StringBuilder, it is just a unit string that is "char" so it is better to use this way. Anyway the output is string after combining.